file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs

```diff
@@ -351,17 +351,17 @@ private void ValidateViewModel()
             {
                 foreach (var file in Files)
                 {
-                    string? directory = Path.GetDirectoryName(file.RawSearchPath);
+                    string? directory = Path.GetDirectoryName(file.SearchPath);
 
                     if (string.IsNullOrWhiteSpace(directory))
                     {
-                        job.ErrorMessages.Add($"Directory not specified or invalid directory specified - '{file.RawSearchPath}'");
+                        job.ErrorMessages.Add($"Directory not specified or invalid directory specified - '{file.SearchPath}'");
```

Comment:
Could it be useful here to indicate both the raw and macro-expanded paths here? Raw helps the user find the right bit in the config file, expanded helps them see what's actually happening on disc?

Reply:
As requested, I have adjusted the error messages to include both the raw and expanded paths